scripts/rollup/bundles.js

```diff
@@ -434,19 +434,16 @@ const bundles = [
 
   /******* ESLint Plugin for Hooks (proposal) *******/
   {
```

Comment:
any reason for this change? looks like it was intended to fit under the 
`/******* ESLint Plugin for Hooks (proposal) *******/` title

Reply:
Not from me. 
I executed `yarn prettier` as peer pull request guidance and prettier decided to do that.